File: modul 6/python/PRAK604-2410817110008-MuhammadAzmaAlFaqih.py
baris1 = input()
baris2 = input()
karakterhasil = []
bintang = 0
pagar = 0
if len(baris1) != len(baris2):
    print("Panjang kalimat berbeda, pesan palsu")
    exit()
for i in range(len(baris1)):
    if baris1[i] == ' ' and baris2[i] == ' ':
        karakterhasil.append(' ')
        continue
    if baris1[i] == baris2[i]:
        karakterhasil.append('*')
        bintang += 1
    else:
        karakterhasil.append('#')
        pagar += 1
print(*karakterhasil, sep='')
print("* = ", bintang)
print("# = ", pagar)
print("Pesan Asli") if bintang >= pagar else print("Pesan Palsu")

File: modul 6/c/PRAK604-2410817110008-MuhammadAzmaAlFaqih.c
#include <stdio.h>
#include <string.h>

int main(){
  int bintang=0, pagar=0;
  char baris1[100], baris2[100];
  scanf("%[^\n]", baris1);
  getchar();
  scanf("%[^\n]", baris2);
  char hasil[strlen(baris1)];
  if (strlen(baris1) != strlen(baris2)){
    printf("Panjang kalimat berbeda, pesan palsu\n");
    return 0;
  }
  for(int i = 0; i < strlen(baris1); i++){
    if(baris1[i] == ' ' && baris2[i] == ' ') {hasil[i] = ' '; continue;}
    if(baris1[i] == baris2[i]) {hasil[i] = '*'; bintang++;}
    else {hasil[i] = '#'; pagar++;}
  }
  for(int i=0; i < strlen(hasil); i++){
    printf("%c", hasil[i]);
  }
  printf("\n* = %d", bintang);
  printf("\n# = %d\n", pagar);
  (bintang >= pagar) ? printf("Pesan Asli\n") : printf("Pesan Palsu\n");
}